.github/ISSUE_TEMPLATE/code_quality.md

```diff
@@ -21,8 +21,3 @@ title: "<Replace this with the actual title>"
 
 <!-- Add any other context suggestion here. -->
 
```

Comment:
Please also remove this blank line so that the lint check can pass. All files should end with a single newline.